### PR TITLE
data: Abhishek

### PR DESCRIPTION
**Pull request by bobbyy16**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3785"></a>

### CODE REVIEW

File: data/Abhishek.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "Abhishek",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Exploring open-source and devOps",
+  "avatar": "https://github.com/bobbyy16.png"
```
Comment (kumarsonsoff3) [inline data/Abhishek.json:6]: There is a missing comma here `,` required to validate your json. 🛠

```suggestion
  "avatar": "https://github.com/bobbyy16.png",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please rename your filename to `username.json`, else some features might not work for you. Plus make the changes suggested below. 🛠

And your filename should be `bobbyy16.json`.

File: data/Abhishek.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      "name": "Youtube",
+      "url": "https://www.youtube.com/channel/UCzycF2lec_1EXj7AQezpboQ",
+      "icon": "FaYoutube"
+    }, 
+
+
+    {
+      "name": "Hashnode",
+      "url": "https://abhishek-devops-engineer.hashnode.dev/",
+      "icon": "FaHashnode"
```
[the remark this answers is not in the document]
Comment (bobbyy16) [reply data/Abhishek.json:26]: Thankyou @kumarsonsoff3

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good, Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**